spec/models/customer_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe Customer, type: :model do
+  describe 'relationships' do
+    it { should have_many :items }
+    it { should have_many :customer_items }
+    it { should have_many :items }
+    it { should belong_to :supermarket }
```

Comment:
Customer does *not* need to belong to the supermarket. The spec just indicates that Items belong to a supermarket but customers don't need to